src/server/api/controllers/vessel-reports.controller.js

```diff
@@ -74,8 +74,28 @@ const getVesselReportById = async id => {
   }
 };
 
+//Get Suggested-route by vessel-report-id
+const getsuggestedRouteByVesselReportId = async id => {
```

Comment:
Looks good to me but only one comment. Can you rename the function name to "getSuggestedRouteByVesselReportId"

Reply:
Agree. Please fix this.

Reply:
fixed